### PR TITLE
added GetUserByIdQuery and Handler

### PR DESCRIPTION
**Pull request by kevjo15**
# Pull Request Title

GetUserByIdQuery and Handler

## Description

Added GetUserByIdQuery and GetUserByIdQueryHandler

## Changes Made

Added GetUserByIdQuery and GetUserByIdQueryHandler

## Related ticket or issue
#60 

## Checklist

- [x] Code adheres to the coding conventions
- [] Unit tests are added for new code
- [] Existing unit tests are updated if needed
- [x] Code follows the principles of Clean Architecture
- [x] Code formatting complies with the `.editorconfig` file
- [x] Pull request template is filled out

### CODE REVIEW

File: Application-Layer/Queries/GetUserById/GetUserByIdQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        public GetUserByIdQueryHandler(UserManager<IdentityUser> userManager)
+        {
+            _userManager = userManager;
+        }
+
+        public async Task<IdentityUser> Handle(GetUserByIdQuery request, CancellationToken cancellationToken)
+        {
+            if (string.IsNullOrWhiteSpace(request.UserId))
+            {
+                throw new ArgumentException("UserId får inte vara tomt.");
```
Comment (Nemanja1208) [inline Application-Layer/Queries/GetUserById/GetUserByIdQueryHandler.cs:23]: I think we should keep the app and code in English broski...

**Review by Nemanja1208 — COMMENTED** [review]
Small comment

File: Application-Layer/Queries/GetUserById/GetUserByIdQueryHandler.cs (hunk 1 above)
Comment (kevjo15) [reply Application-Layer/Queries/GetUserById/GetUserByIdQueryHandler.cs:23]: Alright im on it!

**Review by ElliotDahlin — APPROVED** [review]
Very nice

**Pull request merged**